dapr/proto/runtime/v1/dapr.proto

```diff
@@ -595,4 +605,35 @@ message UnlockResponse {
   }
 
   Status status = 1;
-}
\ No newline at end of file
+}
+
+message WorkflowReference {
+  string InstanceID = 1;
+}
+
+message GetWorkflowRequest {
+  WorkflowReference WorkflowInfo  = 1;
```

Comment:
Maybe call the property `WorkflowReference` to match the type? `WorkflowInfo` sounds too generic, as if it contains a lot more information about the workflow than just the instance ID.
```suggestion
  WorkflowReference WorkflowReference  = 1;
```